gamification/process/update.py

```diff
@@ -135,6 +136,34 @@ def get_participant_objects():
     return participants
 
 
+def get_participant_objects_meta_review():
+    meta_review_participant = meta_review_participants.objects.all()
+    return meta_review_participant
+
+
+def update_participants_data_with_meta_review(participant):
+    """
+    Update participants based on meta-review
+
+    This method updates every participant based on the meta-review
+    received or given. If the meta-review is completed then it calls
+    'Completed Meta-Review' activity.
+    """
+    pos_in_meta_review = meta_review_participants.objects.get(
+                        pos_in=meta_review_participants.pos_in)
```

Comment:
I don't quite understand this. If you want to get the value of `pos_in` of the current meta-review participant (seems the parameter of this function is meta_review participant), you should do `pos_in_meta_review = participant.pos_in`